.github/workflows/release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup kustomize
+        uses: imranismail/setup-kustomize@v2
+
+      - name: Generate release manifests
+        run: |
+          make manifests
+          kustomize build ./manifests > ./manifests/vm-console-proxy.yaml
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Upload generated release manifests
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: manifests/vm-console-proxy.yaml
+          asset_name: vm-console-proxy.yaml
+          asset_content_type: text/plain
+
+      - name: Commit generated release manifests
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: "chore: update kubernetes manifests to ${{ steps.get_release.outputs.tag_name }}"
+          default_author: github_actions
+          commit: --signoff
+          new_branch: "vm-console-proxy-${{ steps.get_release.outputs.tag_name }}"
+
+      - name: Open pull request
```

Comment:
Can you explain why are you creating a new PR for this repository?

Reply:
This PR will be update the `./manifests/vm-console-proxy.yaml`, similar to what @ksimon1 did in kubevirt-tekton-tasks [1].

We don't need it, and it will be updated manually by the developer?

[1] https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/.github/workflows/release.yaml#L98

Reply:
Why do we need the file `./manifests/vm-console-proxy.yaml` ?

Reply:
Alright, then we'll just generate that file in SSP-operator repository.